unlock-app/src/hooks/useFiatKeyPrices.ts

```diff
@@ -62,7 +62,7 @@ export const useFiatKeyPrices = (lockAddresses: string[]) => {
 
   useEffect(() => {
     lockAddresses.forEach(address => getFiatKeyPriceFor(address))
-  }, [lockAddresses])
+  }, [...lockAddresses])
```

Comment:
You're not supposed to use a spread element in a dependency array, it interferes with React's ability to properly determine what changed

Reply:
then let's do a `JSON.stringify(lockAddresses)`